startup.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+npm install
```

Comment:
`yarn` would be better I suppose. I don't know if that's installed in the VM though..

Reply:
Done 👍